app/setup/styleguide.routes.js

```diff
@@ -113,6 +114,14 @@ export default [
       title: 'Swiper',
     }
   },
+  {
+    path: `${root}/swiper-modal`,
+    name: 'styleguideSwiperModal',
+    component: swiperModal,
+    meta: {
+      title: 'Swiper Modal',
+    }
+  },
```

Comment:
I would not use a separate page for this component because it can be tested on the product-detail page. But it's a general question, I think we should only add example pages and not a page for each component.

Reply:
I have removed the Swiper Modal route, especially since we already have a swiper route as well. Maybe this one can be deleted as well, i left it in for now though.